tests/unit/amo/components/TestAddon.js

```diff
@@ -866,34 +866,52 @@ describe(__filename, () => {
   });
 
   it('renders a static theme preview as an image', () => {
+    const headerImageThumb = 'https://addons.cdn.mozilla.net/thumb/1.png';
     const root = shallowRender({
       addon: createInternalAddon({
         ...fakeTheme,
         type: ADDON_TYPE_STATIC_THEME,
+        previews: [
```

Comment:
It's a bit annoying that you have to specify all of this, but I think it's correct to pass valid values into the component. You could save yourself from having to do it here by creating a new object in `tests/unit/amo/helpers.js` for `fakePreview`, and then using that to populate `fakeAddon` in the same file, and also using `fakePreview` here and only overriding the one property the test cares about.

Reply:
Yeh I like this and will this out, thanks for suggestion : )